src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,166 @@
+  <string name="project_website">Sitio web del projecto</string>
+  <string name="promotes_non_free_network_services">Promueve servicios de red no libres</string>
+  <string name="promotes_non_free_software">Promueve software no libre</string>
+  <string name="provided_by_FORMAT">Proporcionado por %s</string>
+  <string name="proxy">Proxy</string>
+  <string name="proxy_host">Servidor de proxy</string>
+  <string name="proxy_port">Puerto de proxy</string>
+  <string name="proxy_type">Tipo de proxy</string>
+  <string name="repositories">Repositorios</string>
+  <string name="repository">Repositorio</string>
+  <string name="repository_not_used_DESC">Este repositorio todavía no ha sido usado. Necesita activarlo para ver
+    las aplicaciones que proporciona.</string>
+  <string name="repository_unsigned_DESC">No firmado. No se pudo verificar la lista de aplicaciones. Tenga cuidado al descargar
+    aplicaciones de repositorios no firmados.</string>
+  <string name="requires_FORMAT">Requiere %s</string>
+  <string name="save">Guardar</string>
+  <string name="saving_details">Guardando detalles</string>
+  <string name="screenshots">Capturas de pantalla</string>
+  <string name="search">Buscar</string>
+  <string name="select_mirror">Seleccione un espejo</string>
+  <string name="show_more">Mostrar más</string>
+  <string name="show_older_versions">Mostrar versiones antiguas</string>
+  <string name="signature_FORMAT">Firma %s</string>
+  <string name="signed_using_unsafe_algorithm">Firmado con un algoritmo inseguro</string>
+  <string name="skip">Saltar</string>
+  <string name="socks_proxy">Proxy SOCKS</string>
+  <string name="sorting_order">Orden de clasificación</string>
+  <string name="source_code">Código fuente</string>
+  <string name="source_code_no_longer_available">El código fuente ya no está disponible</string>
+  <string name="suggested">Sugerido</string>
+  <string name="sync_repositories">Sincronizar repositorios</string>
+  <string name="sync_repositories_automatically">Sincronizar repositorios automáticamente</string>
+  <string name="syncing">Sincronizando</string>
+  <string name="syncing_FORMAT">Sincronizando %s</string>
+  <string name="system">Sistema</string>
+  <string name="tap_to_install_DESC">Pulse para instalar.</string>
+  <string name="theme">Tema</string>
+  <string name="tracks_or_reports_your_activity">Rastrea o informa de su actividad</string>
+  <string name="uninstall">Desinstalar</string>
+  <string name="unknown">Desconocido</string>
+  <string name="unknown_error_DESC">Error desconocido.</string>
+  <string name="unknown_FORMAT">Desconocido: %s</string>
+  <string name="unsigned">No firmado</string>
+  <string name="unstable_updates">Actualizaciones inestables</string>
+  <string name="unstable_updates_summary">Sugerir instalar versiones inestables</string>
+  <string name="unverified">No verificado</string>
+  <string name="update">Actualizar</string>
+  <string name="updates">Actualizaciones</string>
```

Comment:
This is the direct translation, but it's so long it barely fits in its own container in the current UI. Considering changing it to the much shorter "Actualizar" as in the verb "Update", but I will wait for feedback first.